src/main/frontend/components/repo.cljs

```diff
@@ -269,7 +268,7 @@
              (let [repo-path (get-repo-name current-repo)
                    short-repo-name (if (or (util/electron?)
                                            (mobile-util/is-native-platform?))
-                                     (shorten-repo-name repo-path)
+                                     (last (string/split repo-path #"/"))
```

Comment:
This fixes displayed graph name mentioned in https://github.com/logseq/logseq/pull/3368#issuecomment-987903763

Reply:
**nit/non-blocking suggestion:** It could be nice to extract this as a helper function, similar to how we have `text/get-graph-name-from-path`. I imagine this will be used again in the future.

Reply:
It is a revert to the original code.
After searching the codebase, I find the following functions:

- `text/get-file-basename`
- `util/get-dir-and-basename`

Both fns can be used here. 😆  Will write a helper function.